var/spack/repos/builtin/packages/doxygen/package.py

```diff
@@ -30,3 +31,6 @@ class Doxygen(CMakePackage):
 
     # optional dependencies
     depends_on("graphviz", when="+graphviz", type='run')
+
+    # Support C++14's std::shared_ptr
+    patch('shared_ptr.patch', when='@1.8.14')
```

Comment:
Can you add a reference to https://github.com/Sleepyowl/doxygen/commit/6c380ba91ae41c6d5c409a5163119318932ae2a3 in the comment above? That would probably be a convenient way for anybody to check where the patch is coming from.

Reply:
Fixed in 4bd4319.